manager/manager.py

```diff
@@ -570,6 +571,41 @@ def generate_dashboard(self):
                 str(self.jobs_total) + '</td></tr>',
                 'html.parser'))
 
+            # build job failures table
+
+            jobFailures = self.get_job_failures(totalJobs)
+            body.append(BeautifulSoup('<h2>Failed Jobs</h2>', 'html.parser'))
+            failureTable = BeautifulSoup('<table id="failed" cellpadding="4" border="1" style="border-collapse:collapse"></table>','html.parser')
+            failureTable.table.append(BeautifulSoup(
+                '<tr id="header"><th class="hdr">Time</th><th class="hdr">Errors</th><th class="hdr">Source</th><th class="hdr">Destination</th><th class="hdr">Job ID</th></tr>',
+                'html.parser'))
+
+            for i in range(0, MAX_FAILURES):
+                if i >= len(jobFailures):
+                    break
+
+                item = jobFailures[i]
+
+                identifier = item['identifier']
+                identifiers = identifier.split("/")
+                sourceSubPath = "/".join(identifiers[:2])
+                gogs_url = self.gogs_url
+                if gogs_url == None :
+                    gogs_url = 'https://git.door43.org'
+
+                sourceUrl = gogs_url + "/" + sourceSubPath
+                destinationUrl = 'https://live.door43.org/u/' + identifier
```

Comment:
You shouldn't be hard coding the destination domain name, but determine it from the URL that called api.door43.org/dashboard. That determines what database to read and thus what door43.org site the converted files are on. You shouldn't hard code the destination URL as then we won't be able to test it when we go to https://test-api.door43.org/dashboard and https://dev-api.door43.org/dashboard. You can also determine it from the api_url or  cdn_url passed to the dashboard lambda function from the API gateway. That's what I often do. See the door43_deploy lambda function for how this is done.